docs/features/icons.stories.mdx

```diff
@@ -4,10 +4,8 @@ import { Meta, Intro, Icons } from '../../.storybook/components';
 
 # Icons
 
-<Intro as="div">
-
-The icons used throughout Circuit UI come from SumUp's icon library, [`@sumup/icons`](Packages/icons), which is a required peer dependency of `@sumup/circuit-ui`. Refer to its [documentation](Packages/icons) for usage instructions.
```

Comment:
👀 Why this change?

Reply:
Ah I was trying to fix the Intro component after I refactored it (because it was hardcoding cascading typography values), but forgot to add this doc back. Thanks! Will link to the docs, though probably out of the `Intro` to avoid having to do the same thing as before

Reply:
Close enough without hacky CSS, will move forward (happy to think of alternatives later if we want to, right now I'd favor simplicity)

<img width="1248" alt="Screenshot 2022-01-28 at 22 00 08" src="https://user-images.githubusercontent.com/35560568/151620828-407d1448-5674-4250-bd05-39d3d9b177f8.png">